spec/javascripts/pageflow/ui/utils/i18n_utils_spec.js

```diff
@@ -31,6 +33,33 @@ describe('pageflow.i18nUtils', function() {
 
       expect(result).to.eq('Value interpolated');
     });
+
+    it('escapes html', function() {
+      var result = pageflow.i18nUtils.findTranslation(
+        ['html.key.without.suffix']
+      );
+
+      expect(result).to.eq('&lt;div /&gt;');
```

Comment:
This is an unwanted breaking change imo. Escaping should only take place, if the caller signals via the `html` option that he intents to treat the result as HTML. If we expect text, we never want to see HTML entities in the result.

If I read the implementation correctly the default value is also not escaped, which I think is correct behavior. We should also have tests for the treatment of the default value.